python/py-plotly/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-plotly
+version             5.3.1
+revision            0
+
+categories          python graphics
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {nist.gov:joe.fowler @joefowler} openmaintainer
+
+description         Interactive graphing library for Python.
+long_description \
+    Plotly's Python graphing library makes interactive, publication-quality graphs. Examples of how to \
+    make line plots, scatter plots, area charts, bar charts, error bars, box plots, histograms, \
+    heatmaps, subplots, multiple-axes, polar charts, and bubble charts. 
+
+homepage            https://plotly.com/python/
+
+checksums           rmd160  679e2ce0095eb3b9e250ed2d6b100120a4613bc1 \
+                    sha256  6598393e898a9c5ae78397f76f07002ec41fd92e5f746d3b9806248d53885643 \
+                    size    7296601
+
+python.versions     36 37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+        port:py${python.version}-setuptools
+    
+    depends_lib-append \
+        port:py${python.version}-matplotlib \
+        port:py${python.version}-six
```

Comment:
it looks like a dependency is missing here, see: https://github.com/plotly/plotly.py/blob/b763ef96cc36f460d4096650e54f51bf85886cb8/packages/python/plotly/setup.py#L552 ?

Reply:
Good catch. Thank you. Updated commits coming shortly....